tools/nl/embeddings/build_embeddings.py

```diff
@@ -42,23 +42,16 @@
 flags.DEFINE_string('bucket_name_v2', 'datcom-nl-models', 'Storage bucket')
 flags.DEFINE_string('embeddings_size', '', 'Embeddings size')
 
-flags.DEFINE_string('local_sheets_csv_filepath',
-                    'data/curated_input/sheets_svs.csv',
-                    'Local Sheets csv (relative) file path')
-flags.DEFINE_string(
-    'sheets_url',
-    'https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU',
-    'Google Sheets Url for the latest SVs')
-flags.DEFINE_list(
-    'worksheet_names', ['Demo_SVs', 'MAIN_DC_SDG_GOALS_INDICATORS_ONLY'],
-    'List of names of worksheets in the Google Sheets file to use')
+flags.DEFINE_string('curated_input_path',
+                    'data/curated_input/main_sheets_svs.csv',
```

Comment:
super nit: feel free to ignore or defer.  like we have data/alternatives/main|sdg|xyz, we could have sub-dirs in data/curated_input/

Reply:
done